spec/linter-eslint-spec.js

```diff
@@ -213,6 +215,46 @@ describe('The eslint provider for Linter', () => {
     })
   })
 
+  describe('when a file is not specified in .eslintignore file', async () => {
+    it('will give warnings when linting the file', async () => {
+      const tempPath = await copyFileToTempDir(path.join(eslintIgnoreDir, 'ignored.js'))
+      const tempDir = path.dirname(tempPath)
+
+      const editor = await atom.workspace.open(tempPath)
+      atom.config.set('linter-eslint.disableEslintIgnore', false)
+      await copyFileToDir(path.join(eslintIgnoreDir, '.eslintrc.yaml'), tempDir)
+
+      const messages = await lint(editor)
+      expect(messages.length).toBe(1)
+      expect(editor)
```

Comment:
What's this line for?

Reply:
Hahaha. I really have no idea. I can only assume it is remnant from some sort of rewrite/copy-paste something or other that I didn't notice since it doesn't break anything. Will 🔥